hare/broker.go

```diff
@@ -57,13 +57,13 @@ type Broker struct {
 	inbox          chan service.GossipMessage
 	queue          priorityq.PriorityQueue
 	queueChannel   chan struct{} // used to synchronize the message queues
-	syncState      map[instanceID]bool
```

Comment:
Are we sure we want to eliminate the hare instanceID type entirely? @noamnelke what do you think?

Reply:
this type didn't provide any utility, the only method that it had was Bytes() and it was doing exactly the same thing as LayerID.Bytes(). but now I also had to cast it to LayerID every time I had to modify it, and then back to instanceID to store it

Reply:
I'm with @dshulyak on this. I've hated that type for ages and wanted to eliminate it, too. Other than adding unjustified code complexity, I find that it complicates the mental model of what the Hare is doing. Fewer concepts to keep in mind == better.